examples/00-EDB/05_Plot_nets.py

```diff
@@ -1,65 +1,69 @@
 """
-EDB: plot nets with Matplotlib
-------------------------------
-This example shows how you can use the ``Edb`` class to plot a net or a layout.
-"""
+# EDB: plot nets with Matplotlib
 
-###############################################################################
-# Perform required imports
-# ~~~~~~~~~~~~~~~~~~~~~~~~
-# Perform required imports, which includes importing a section.
+This example shows how to use the ``Edb`` class to view nets, layers and
```

Comment:
```suggestion
This example shows how to use the ``Edb`` class to view nets, layers, and
```
We use Oxford commas, which simply means always put a comma before the "and" or the "or" in a list of items.